### ISSUE TITLE
Newlines (\n) in ARG_HELP 2nd argument should start a new line.

### ISSUE THREAD

**Issue by avysharell** [matejak/argbash#65]
In the 2nd Argument to ARG_HELP it might be helpfull to embed newlines to create paragraphs in the help text at the end of the help message. In the current version these are printed as backslahs+n ("\n") instead of starting a new line.

**Comment by avysharell** [comment]
After some testing but without deeper understanding I think changing line 362 in stuff.m4, function
m4_define([_MAKE_HELP], [MAKE_FUNCTION( ... )
from
```
    [m4_ifnblank(m4_quote(_HELP_MSG_EX), m4_dquote(_INDENT_()[printf '\n%s\n' "]_HELP_MSG_EX"_ENDL_()))],
```
to
```
    [m4_ifnblank(m4_quote(_HELP_MSG_EX), m4_dquote(_INDENT_()[printf '\n%b\n' "]_HELP_MSG_EX"_ENDL_()))],
```
would implement this. If it helps I can branch and create a pull-request for this simple change
(if i'm at all correct).

**Comment by matejak** [comment]
Thank you for your feature request, I will take a look into this. Although your suggestion would work, there is already a `\n` -> newline substitution implemented elsewhere in the code, so I lean towards using that path.

**Comment by accetto** [comment]
I've found that the Docker image published on Docker Hub still does not include this enhancement. Could you update it please?
Very nice work otherwise.

**Comment by matejak** [comment]
This will be released as 2.8.1, and I expect to release it some time next week.
